internal/pkg/state/local/create.go

```diff
@@ -19,11 +19,18 @@ func (m *Manager) createObject(key model.Key, name string) (model.Object, error)
 		if err != nil {
 			return nil, err
 		}
-		return &model.Config{
+		config := &model.Config{
 			ConfigKey: k,
 			Name:      name,
 			Content:   content,
-		}, nil
+		}
+		if component.IsTransformation() {
+			config.Transformation = &model.Transformation{}
+		}
+		if component.IsOrchestrator() {
+			config.Orchestration = &model.Orchestration{}
+		}
```

Comment:
Pri `kbc create config` musime inicializovat `.Transformation`, `.Orchestration` fields.
Je to nedostatok aktualneho mappingu, kde nemame objekt typu `Transfromation` alebo `Orchestration`, ale iba `Config` vo vseobecnosti, preto je to potrebne rucne osetrit.